src/univariates.jl

```diff
@@ -182,7 +182,8 @@ std(d::UnivariateDistribution) = sqrt(var(d))
 """
     median(d::UnivariateDistribution)
 
-Return the median value of distribution `d`.
+Return the median value of distribution `d`. The median is the smallest `x` such that `cdf(d, x) ≥ 1/2`.
```

Comment:
Can an informal description be added before the formal definition to make it easier to understand? 

Something like "Informally, the median is the value which divides the distribution `d` in two equally sized portions. Formally, the median is the smallest `x` such that `cdf(d, x) ≥ 1/2`."

Reply:
This is a can of worms, let's avoid explaining medians, people will know to look these things up.